### ISSUE TITLE
Can we use Adam to train CTC and Transducer? 

### ISSUE THREAD

**Issue by Marcovaldong** [SeanNaren/deepspeech.pytorch#439]
I'm training CTC and RNN-Transducer. I tried to train CTC with Adam and SGD. Usually, Adam makes the model converges more quickly. However, I trained the CTC with Adam for several epochs, the CER is still high. 

I noticed #407, but I think Adam dosen't influences the choice of hyper-parameters.

So, can we use Adam to train CTC, RNN-Transducer or other ASR models? 

@SeanNaren

**Comment by Kabur** [comment]
Might be the learning rate. Adam needs a different one than SGD, start with 0.001.

**Comment by Marcovaldong** [comment]
@Kabur In my experiments, strating with 0.0003 is ok.
